pythran/pythonic/include/numpy/ufunc_accumulate.hpp

```diff
@@ -0,0 +1,26 @@
+#ifndef UFUNC_NAME
+#error missing UFUNC_NAME
+#endif
+
+// clang-format off
+#include INCLUDE_FILE(pythonic/include/numpy,UFUNC_NAME)
+// clang-format on
```

Comment:
It doesn't detect we are performing an include and add spaces around "\" which give a wrong path